website/content/docs/auth/kubernetes.mdx

```diff
@@ -322,6 +322,71 @@ The Kubernetes Auth Plugin has a full HTTP API. Please see the
 
 [k8s-tokenreview]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#tokenreview-v1-authentication-k8s-io
 
+## Workflows
+
+Refer to the following workflow examples for Kubernetes auth method usage:
+
+### Working with templated policies
+
+Set `use_annotations_as_alias_metadata=true` in your Kubernetes auth
+configuration to use Kubernetes Service Account annotations for
+[Vault alias](/vault/docs/concepts/identity#entities-and-aliases) metadata.
+
+When `use_annotations_as_alias_metadata` is true, you can use the
+`identity.entity.aliases.<mount accessor>.metadata.<metadata key>` template
+parameter when you create [templated policies](/vault/docs/concepts/policies#templated-policies).
+
+To use annotations as alias metadata, you must give Vault permission to read
+service accounts from the Kubernetes API.
+
+#### Scenario Introduction
+
+Assume you have the following policy requirement:
+
+Applications can perform read operations on its allocated key/value secret path:
+`(env-kv/data/<env>)`
+
+#### Annotate Kubernetes Service Accounts with their dedicated secret paths
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: app
+  namespace: demo
+  annotations:
+    vault.hashicorp.com/alias-metadata-env: demo/app
+```
+
+When the application, `app`, logs in with JWT for the service account, Vault
+renders the alias metadata as `env : demo/app`.
+
+#### Create a templated ACL policy
+
+The `env-tmpl` policy lets applications read their secrets defined in KV v2
+secret engine. So you can read the example mount accessor value
+(`auth_kubernetes_bcecb1e1`) from the `sys/auth` endpoint.
```

Comment:
Just thinking this could be a little more direct, and link to sys/auth and "vault auth list".

```suggestion
secret engine. Use the mount accessor value
(`auth_kubernetes_bcecb1e1`) from the [`sys/auth`](https://developer.hashicorp.com/vault/api-docs/system/auth) endpoint or the [`vault auth list`](https://developer.hashicorp.com/vault/docs/commands/auth/list) command.
```

Reply:
And oops, those should be relative links (`/vault/api-docs/system/auth`, `/vault/docs/commands/auth/list`)